test_user_management.py

```
@@ -0,0 +1,391 @@
+        return hashlib.md5(token.encode()).hexdigest()
+    
+    def validate_session_token(self, token: str) -> Optional[int]:
+        """Validate session token"""
+        try:
+            # SECURITY ISSUE 12: Weak token validation
+            parts = token.split('_')
+            if len(parts) >= 2:
+                user_id = int(parts[0])
+                return user_id
+            return None
+        except:
```

Ruff (0.15.6)

[error] Do not use bare `except`

(E722)